first-year-parse/README.md

```diff
@@ -0,0 +1,13 @@
+1. Convert pdf to .docx and Run textExtract.py
+
+2. In the semData file :
+
+ a. delete all text apart from first year time table data and odd section data
+
+ b. add a  &#39;#&#39; character for every empty slot
+
+ d. duplicate subject and room entries for double lectures in the text file
+
+3. Run the extract script.
```

Comment:
What command is to be used? Please mention that clearly.

Also, is there no `requirements.txt` file for this.

You can create a requirements.txt file by doing `pip3 freeze > requirements.txt`